webapp/nuxt.config.js

```diff
@@ -1,11 +1,13 @@
 export default {
   // Global page headers (https://go.nuxtjs.dev/config-head)
   head: {
-    title: 'nuxt',
+    title: 'HackerNews',
     meta: [
       { charset: 'utf-8' },
       { name: 'viewport', content: 'width=device-width, initial-scale=1' },
-      { hid: 'description', name: 'description', content: '' }
+      { hid: 'description', name: 'description', content: '' },
+      { name: 'theme-color', content: '#2196F3'},
+      { name: 'theme_color', content: '#2196F3'}
```

Comment:
I guess you don't need a duplicate at that point. It works like this theme_color or much better, see https://pwa.nuxtjs.org/manifest/